icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -1131,30 +1131,30 @@ class U_I18N_API Scale : public UMemory {
 
 namespace impl {
 
-// Do not enclose entire Usage with #ifndef U_HIDE_INTERNAL_API, needed for a protected field
+// Do not enclose entire StringProp with #ifndef U_HIDE_INTERNAL_API, needed for a protected field
 /**
  * Manages NumberFormatterSettings::usage()'s char* instance on the heap.
  * @internal
  */
-class U_I18N_API Usage : public UMemory {
+class U_I18N_API StringProp : public UMemory {
```

Comment:
I still feel that `Usage` is more clear than StringProp

Reply:
Usage is the wrong type for unitDisplayCase. This would be very confusing:

    Usage unitDisplayCase;

It isn't a usage. So what is it? It's really just a string container, for a string property of MacroProps.